app/models/mhv_account.rb

```diff
@@ -120,8 +127,12 @@ def create_mhv_account!
 
   def upgrade_mhv_account!
     if may_upgrade?
+      StatsD.increment("#{STATSD_ACCOUNT_UPGRADE_KEY}.total")
+
       client_response = mhv_ac_client.post_upgrade(params_for_upgrade)
       if client_response[:status] == 'success'
+        StatsD.increment("#{STATSD_ACCOUNT_UPGRADE_KEY}.success")
+
```

Comment:
I can add a failure counter for non 155 returns. Then:
Total calls to mhv-upgrade: `mhv.account.upgrade.total`
Accounts upgraded = `mhv.account.upgrade.success`
Calls that resulted in real errors =`mhv.account.upgrade.failure`
Accounts that were already upgraded =`mhv.account.upgrade.total` - `mhv.account.upgrade.success` - `mhv.account.upgrade.failure`